src/App.jsx

```diff
@@ -13,6 +13,7 @@ import { AuthProvider } from './contexts/AuthContext';
 import ImageRadioButtonsInput from './components/ImageRadioButtonsInput';
 import TextImageInput from './components/TextImageInput';
 import CreateProtocolPage from './pages/CreateProtocolPage';
+import SubForm from './components/SubForm';
 
```

Comment:
```suggestion
import CreateProtocolPage from './pages/CreateProtocolPage';

```
- Importações não usadas;